lib/views/splash_page.dart

```diff
@@ -1,12 +1,64 @@
-import 'package:cycle_lock/controllers/splash_controller.dart';
+import 'dart:async';
+
 import 'package:flutter/material.dart';
-import 'package:get/get_state_manager/get_state_manager.dart';
+import 'package:get/get_core/src/get_main.dart';
+import 'package:get/get_navigation/src/extension_navigation.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:lottie/lottie.dart';
+import 'package:flutter/services.dart';
+import 'auth_page.dart';
+import 'package:get/get.dart';
+
+void main() => {runApp(const SplashPage())};
+
+void startTimer() {
+  Timer(const Duration(seconds: 4), () {
+    navigateUser(); //It will redirect  after 4 seconds
+  });
+}
```

Comment:
put the startTimer() inside splashController
All the logics should be present inside controller
only widgets should be present inside view